src/Polly.Specs/PolicyAsyncSpecs.cs

```diff
@@ -194,7 +194,7 @@ public void Executing_the_policy_action_should_throw_when_context_is_null()
                 .Handle<DivideByZeroException>()
                 .RetryAsync((_, __, ___) => { });
 
-            policy.Awaiting(async p => await p.ExecuteAsync(ctx => TaskHelper.EmptyTask, (Context)null))
+            policy.Awaiting(async p => await p.ExecuteAsync(ctx => TaskHelper.EmptyTask, null))
```

Comment:
I prefer to keep this explicit cast `(Context)null` (and other similar ones).

Without it, to understand at a glance why the compiler automatically chooses that overload without ambiguity, the reader would have to have in mind [point 6 of the overload resolution rules](https://docs.microsoft.com/en-us/dotnet/visual-basic/reference/language-specification/overload-resolution), as well as knowing that `Context : IDictionary<string, object>` (which is what happens to make one Execute overload more specific than the other). 

That's not going to be nearly as grokkable for a new reader to the code, than having the explicit cast in place. IE I'm opting to prioritise human readability over compiler readability in this case.